### DISCUSSION TITLE
md 源文件丢失如何找回？是否可以 html 转 markdown

### DISCUSSION THREAD

**Discussion by chenxia31** [hexojs/hexo#5436] [Question / Answer] [answered]
不小心使用 git pull 将远程仓库覆盖本地，之后不小心把本地的.git文件删除，无法回滚。现在唯一幸存的只有生成的 Html 文件，如何转换成为原始的 markdown 的文件

**Comment by uiolee** [comment, marked as answer]
searching `html to markdown` in any search engines will help you
